examples/pcr/extend.c

```diff
@@ -198,13 +200,19 @@ int TPM2_Extend_Test(void* userCtx, int argc, char *argv[])
 /* --- END TPM2.0 PCR Extend example tool -- */
 /******************************************************************************/
 
+#endif /* !WOLFTPM2_NO_WRAPPER */
 
 #ifndef NO_MAIN_DRIVER
 int main(int argc, char *argv[])
 {
-    int rc;
+    int rc = -1;
 
+#ifndef WOLFTPM2_NO_WRAPPER
     rc = TPM2_Extend_Test(NULL, argc, argv);
+#else
+    (void)argc;
```

Comment:
Can you add a printf in these cases so we know its not compiled in because of the missing TPM wrappers?